pcweb/templates/docpage/blocks/code.py

```diff
@@ -5,6 +5,20 @@
 from pcweb import styles
 
 
+def copy_icon(code: str) -> rx.Component:
```

Comment:
With the new refactor the copy_icon seems to be missing for me

Reply:
I was just looking in the learn section, I think here they use the ```python ``` syntax which doesn't use the copy to clipboard, I'll have to fix this in a follow up